### PR TITLE
Update schema notebook to cover DeclarativeSchema

### PR DESCRIPTION
**Pull request by richard-rogers**
## Description

Updates schema example notebook to cover `DeclarativeSchema`

### CODE REVIEW

File: python/examples/basic/Schema_Configuration.ipynb [hunk 1]
```diff
@@ -1,893 +1,2065 @@
...
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HzEKQywx0oqw"
+      },
+      "source": [
+        "Note that existing columns that are not specified in your custom resolver won't have any metrics tracked. In the example above, we added a `col5` column, but since we didn't link any metrics to it, all of the metrics are `NaN`s.\n",
+        "\n",
+        "## Declarative Schema\n",
+        "\n",
+        "In the previous section, we created subclasses of `Resolver` and implemented its `resovle()` method using control flow. The `DeclarativeSchema` allows us to customize the metrics present in a column by simply listing the metrics we want by data type or column name without implementing a `Resolver` subclass.\n",
```
Comment (FelipeAdachi) [inline python/examples/basic/Schema_Configuration.ipynb]: typo: `resovle()`

**Review by FelipeAdachi — DISMISSED** [review]
That's great!
just added a minor comment there, but otherwise LGTM!

File: python/examples/basic/Schema_Configuration.ipynb (hunk 1 above)
Comment (richard-rogers) [inline python/examples/basic/Schema_Configuration.ipynb]:
```suggestion
        "In the previous section, we created subclasses of `Resolver` and implemented its `resolve()` method using control flow. The `DeclarativeSchema` allows us to customize the metrics present in a column by simply listing the metrics we want by data type or column name without implementing a `Resolver` subclass.\n",
```

**Review by jamie256 — APPROVED** [review]
LGTM!

**Pull request merged**